lib/actions/user.js

```diff
@@ -156,35 +156,52 @@ export function fetchAuth0Token(auth0, intl) {
  */
 function convertRequestToSearch(config) {
   return function (tripRequest) {
-    const { dateCreated, id, requestParameters = {} } = tripRequest
+    const { dateCreated, id, otp2QueryParams = {} } = tripRequest
     const { host, path } = config.api
     return {
       canDelete: false,
       id,
-      query: planParamsToQuery(requestParameters || {}),
+      query: planParamsToQuery(otp2QueryParams || {}),
       timestamp: dateCreated,
-      url: `${host}${path}/plan?${qs.stringify(requestParameters)}`
+      url: `${host}${path}/plan?${qs.stringify(otp2QueryParams)}`
     }
   }
 }
 
+// TODO move to a utils file
+function areRequestsSame(qp1, qp2) {
+  // Everything except modes.
+  // Exclude banned/preferred/unpreferred at this time.
+  return (
+    qp1.bikeReluctance === qp2.bikeReluctance &&
+    qp1.carReluctance === qp2.carReluctance &&
+    qp1.date === qp2.date &&
+    qp1.departArrive === qp2.departArrive &&
+    qp1.from?.name === qp2.from?.name &&
+    qp1.departArrive === qp2.departArrive &&
+    qp1.time === qp2.time &&
+    qp1.to?.name === qp2.to?.name &&
+    qp1.walkReluctance === qp2.walkReluctance &&
+    qp1.walkSpeed === qp2.walkSpeed &&
+    qp1.wheelchair === qp2.wheelchair
+  )
+}
```

Comment:
Instead of doing this many checks, would it be possible to filter out the properties you don't want to compare (eg. modes and banned routes) and then compare using `JSON.stringify`?